modules/navier_stokes/src/fvkernels/WCNSFV2PMomentumDriftFlux.C

```diff
@@ -26,13 +26,15 @@ WCNSFV2PMomentumDriftFlux::validParams()
   params.addParam<MooseFunctorName>("w_slip", "The slip velocity in the z direction.");
   params.addRequiredParam<MooseFunctorName>("rho_d", "Dispersed phase density.");
   params.addParam<MooseFunctorName>("fd", 0.0, "Fraction dispersed phase.");
+  params.renameParam("fd", "fraction_dispersed", "");
 
   MooseEnum coeff_interp_method("average harmonic", "harmonic");
   params.addParam<MooseEnum>("density_interp_method",
                              coeff_interp_method,
                              "Switch that can select face interpolation method for the density.");
 
-  params.set<unsigned short>("ghost_layers") = 2;
+  // when using the fraction of the disperse phase, somehow
+  params.set<unsigned short>("ghost_layers") = 5;
```

Comment:
we're going to want to look into this

Reply:
this is not coming from my PR either. The initial input file checked in was using density where it should have used phase fraction (see the diff on that file, iirc). once it changes it has this issue

Reply:
I can take a look when I come back though. I was out of ideas so I would learn some new technique

Reply:
this is fixed now